test/README.md

```diff
@@ -76,6 +76,11 @@ You can specify only a subsets of test to run with :
 
 same goes for `TestGitlab` or other methods.
 
+If you need to update the golden files in the e2e test you add the flag
+`-test.update-golden=true` to the go test command line to update the golden
```

Comment:
When I've run this locally I've been getting a warning that `update-golden` is deprecated and `update` is the expected flag

![image](https://github.com/user-attachments/assets/2d57bbde-9f0f-46a8-b96a-4c52829b17fa)

Reply:
Since this is ancillary to the main change of this PR, I think it'd be better to keep your added documentation as is and update all of the docs and automation from `-test.update-golden` to `-update` separately